deployment/kube/stage/frontend.yaml

```diff
@@ -17,6 +19,15 @@ spec:
         tier: frontend
     spec:
       affinity:
+        podAffinity:
```

Comment:
I don't understand what this is doing. Can you please explain or point to some documentation?

Reply:
Actually, I just realised your question was about the podAffinity. 

It's actually wrong it's podAntiAffinity it's to make sure that when it's scale-down it leave the pod on multiple zones.

Reply:
OK, that makes more sense, thanks.